database/migrations/20191023190811_create-tables.js

```diff
@@ -9,16 +9,18 @@ exports.up = function(knex) {
 				.notNullable()
 				.unique();
 			table.string("password").notNullable();
-		})
-
-		.createTable("weddings", table => {
-			table.increments();
 			table
 				.string("slug")
 				.notNullable()
 				.unique();
+			table.string("jwt", 512);
+		})
+
+		.createTable("weddings", table => {
+			table.increments();
 			table.datetime("date");
 			table.string("location");
+			table.string("slug").references("couples.slug");
```

Comment:
Slugs are created in 2 spots but it doesn't look to be used anywhere. Why store 2 references for the couple on the wedding?